tidb-cloud/tidb-cloud-overview-for-analyzing-and-tuning-performance.md

```diff
@@ -0,0 +1,126 @@
+
+![Write hotspot](/media/tidb-cloud/tidb-cloud-troubleshoot-write-hotspot.png)
+
+A read hotspot is generally represented in the heat map as a bright horizontal line, usually a small table with a large number of visits, as shown in the following screen shot.
+
+![Read hotspot](/media/tidb-cloud/tidb-cloud-troubleshoot-read-hotspot.png)
+
+Hover over the highlighted block to see which table or index has high traffic, as shown in the following screen shot.
+
+![Hotspot index](/media/tidb-cloud/tidb-cloud-troubleshoot-hotspot-index.png)
+
+#### Scale out
+
+In the metrics under [Overview](/tidb-cloud/monitor-tidb-cluster.md), you can view storage space, CPU utilization, and TiKV IO rate. If any of them are reaching the upper limit for a long time, it is possible that the current cluster size can not meet the business requirements. It is recommended to contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) to confirm if you need to scale out the cluster.
+
+#### Other issues
+
+If the previous methods can not resolve the performance issue, you can contact the  [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) for help. It is recommended to provide the following information to speed up the troubleshooting process.
+
+- The cluster ID
+- The issue interval and a comparable normal interval
+- The problem phenomenon and expected behavior
+- The business load characteristics, such as read or write ratios and primary behavior
+
+## Summary
+
+In general, you can use the following optimization methods to analyze and resolve performance issues.
+
+| Action | Effect |
+|:--|:--|
+| Prepared plan cache + JDBC | Throughput performance will be greatly improved, latency will be significantly reduced, and the average TiDB CPU utilization will be significantly reduced. |
+| Enable AsyncIO and Raft-engine in TiKV | There will be some improvement in throughput performance. |
+| Clustered Index | Throughput performance will be greatly improved. |
+| Scale out TiDB nodes |Throughput performance will be greatly improved.  |
+| Client-side optimization. Split 1 JVM into 3 | Throughput performance will improve significantly and may further continue to improve throughput capacity if further split. |
+| Limit the network latency between the application and the database | High network latency can lead to decreased throughput and increased latency. |
+
+In the future, TiDB Cloud will introduce more observable data and self-diagnostic services. They will provide you with a more comprehensive understanding of performance metrics and operational advice to improve your experience.
```

Comment:
```suggestion
In the future, TiDB Cloud will introduce more observable metrics and self-diagnostic services. They will provide you with a more comprehensive understanding of performance metrics and operational advice to improve your experience.
```

Reply:
Do we really need this sentence because it is like a future promise? If not, can we remove it?